Makefile

```diff
@@ -1,6 +1,9 @@
 OS=$(shell uname)
 VERSION=v1.12.0
-CGO_LDFLAGS="-Lgen/third_party/rplidar_sdk-release-${VERSION}/sdk/output/${OS}/Release/"
+GIT_REVISION = $(shell git rev-parse HEAD | tr -d '\n')
+TAG_VERSION?=$(shell etc/tag_version.sh)
+CGO_LDFLAGS="-L 'gen/third_party/rplidar_sdk-release-${VERSION}/sdk/output/${OS}/Release/'"
```

Comment:
These link flags need to be kept separate as they are instructions for how CGo sound link with the C FFI Swig generates.